src/codec/MapAddEntryListenerCodec.ts

```diff
@@ -0,0 +1,78 @@
+import {ClientMessage} from "../ClientMessage";
+import {BitsUtil} from '../BitsUtil';
+import {CustomCodec} from './CustomCodec';
+import {MapMessageType} from './MapMessageType';
+
+var REQUEST_TYPE = MapMessageType.MAP_ADDENTRYLISTENER
+var RESPONSE_TYPE = 104
+var RETRYABLE = false
+
+
+export class MapAddEntryListenerCodec{
+
+constructor() {
+}
+
+
+
+
+static calculateSize(name, includeValue, listenerFlags, localOnly){
+    // Calculates the request payload size
+    var dataSize = 0;
+    dataSize += BitsUtil.calculateSizeStr(name);
+    dataSize += BitsUtil.BOOLEAN_SIZE_IN_BYTES;
+    dataSize += BitsUtil.INT_SIZE_IN_BYTES;
+    dataSize += BitsUtil.BOOLEAN_SIZE_IN_BYTES;
+    return dataSize;
+}
+
+static encodeRequest(name, includeValue, listenerFlags, localOnly){
+    // Encode request into clientMessage
+    var payloadSize;
+    var clientMessage = new ClientMessage(payloadSize=this.calculateSize(name, includeValue, listenerFlags, localOnly));
+    clientMessage.setMessageType(REQUEST_TYPE);
+    clientMessage.setRetryable(RETRYABLE);
+    clientMessage.appendStr(name);
```

Comment:
ClientMessage api is not compatible with the generated code. Please validate them. 
Example: appendStr --> appendString  etc